### ISSUE TITLE
Number alignment 

### ISSUE THREAD

**Issue by IanWorthington** [vdmeer/asciitable#25]
Can the library handle correctly aligning numbers in columns, ie all numbers should have their units value  vertically aligned?

**Comment by marvk** [comment]
I agree, and on that note setting alignment for rows rather than columns would be quite helpful.
